pkg/cosign/fetch.go

```diff
@@ -78,11 +78,12 @@ func FetchSignaturesForReference(ctx context.Context, ref name.Reference, opts .
 		return nil, fmt.Errorf("no signatures associated with %v: %w", ref, err)
 	}
 
-	g := pool.New(runtime.NumCPU())
 	signatures := make([]SignedPayload, len(l))
+	var g errgroup.Group
+	g.SetLimit(runtime.NumCPU())
```

Comment:
Can you file an issue on PKG to deprecate `pool` in favor of this, and possibly move to this model for its implementation?

Reply:
https://github.com/knative/pkg/issues/2556